pkg/search/proxy/controller.go

```diff
@@ -182,6 +182,12 @@ func (ctl *Controller) reconcile(util.QueueKey) error {
 			if !util.ClusterMatches(cluster, registry.Spec.TargetCluster) {
 				continue
 			}
+
+			if !util.IsClusterReady(&cluster.Status) {
+				klog.Warningf("cluster %s is notReady", cluster)
```

Comment:
klog.Warningf("cluster %s is notReady", cluster.Name)?

Reply:
i fix it [https://github.com/karmada-io/karmada/pull/3010/commits/087aecdc37939684235fded7236efb5e9b7ab17c](url)